src/arc/mp/MultiplayerChat.as

```diff
@@ -271,70 +271,26 @@ package arc.mp
 
         public static function textFormatLevel(user:Object):String
         {
-            var divisionColor:Array = ArcGlobals.divisionColor;
-            var divisionTitle:Array = ArcGlobals.divisionTitle;
-            var color:int;
-            var division:int;
-            var title:String;
-            if (user.userLevel > 121)
-            {
-                color = divisionColor[8];
-                title = divisionTitle[8];
-                division = 9;
-            }
-            else if (user.userLevel >= 101)
-            {
-                color = divisionColor[7];
-                title = divisionTitle[7];
-                division = 8;
-            }
-            else if (user.userLevel >= 94)
-            {
-                color = divisionColor[6];
-                title = divisionTitle[6];
-                division = 7;
-            }
-            else if (user.userLevel >= 83)
-            {
-                color = divisionColor[5];
-                title = divisionTitle[5];
-                division = 6;
-            }
-            else if (user.userLevel >= 69)
-            {
-                color = divisionColor[4];
-                title = divisionTitle[4];
-                division = 5;
-            }
-            else if (user.userLevel >= 59)
-            {
-                color = divisionColor[3];
-                title = divisionTitle[3];
-                division = 4;
-            }
-            else if (user.userLevel >= 50)
-            {
-                color = divisionColor[2];
-                title = divisionTitle[2];
-                division = 3;
-            }
-            else if (user.userLevel >= 26)
-            {
-                color = divisionColor[1];
-                title = divisionTitle[1];
-                division = 2;
-            }
-            else
+            const divisionColor:Array = ArcGlobals.divisionColor;
+            const divisionTitle:Array = ArcGlobals.divisionTitle;
+            const divisionLevel:Array = ArcGlobals.divisionLevel;
+            const level:int = user.userLevel;
+
+            var i:int;
```

Comment:
Please update this `i` variable to be more descriptive, especially since it breaks out of a loop context.

Reply:
Renamed `i` to `div`, and moved the logic for getting the division colours and titles into ArcGlobals.